validator/main.go

```diff
@@ -649,7 +650,7 @@ func setEnvVar(envvars []string, key, value string) []string {
 // For driver container installs, check existence of .driver-ctr-ready to confirm running driver
 // container has completed and is in Ready state.
 func assertDriverContainerReady(silent bool) error {
-	command := "bash"
+	command := "sh"
```

Comment:
can we move this to constant? Let's call it `shell`

Reply:
Updated.